windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -596,12 +596,12 @@
   <!-- Context: Configuration Dialog - Update tab pop up -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 18.0.168 -->
-  <string name="S_Update_Restart_Req" comment="Update tab pop up - warns user restart will require restart ">Updating now will require a computer restart.</string>
+  <string name="S_Update_Restart_Req" comment="Update tab pop up - warns user restart will require restart ">Applying the update now will require a restart of your computer before you can use Keyman again. Update now anyway?</string>

```

Comment:
This text in the dialog feels a bit disconnected -- like we need an intro sentence?

"An updated to Keyman is read to install.

Applying the update now will require a restart ...etc"

Reply:
Is the colon after "restarted" (in the first image) a change that's introduced in a different PR? Because in this file it has a period (line 594).

Reply:
Ah, I see. It's in #13555 - I just didn't look far enough in that PR :smile: